website/docs/d/eventhub_namespace.html.markdown

```diff
@@ -42,6 +42,8 @@ output "eventhub_namespace_id" {
 
 * `maximum_throughput_units` -  Specifies the maximum number of throughput units when Auto Inflate is Enabled.
 
+* `zone_redundant` - Enabling this property creates a Standard Event Hubs Namespace in regions supported availability zones.
```

Comment:
since this is a Data Source (e.g. retrieving a read-only value) we can make this:

```suggestion
* `zone_redundant` - Is this EventHub Namespace deployed across Availability Zones?
```